src/osx.md

```diff
@@ -1,12 +1,17 @@
 # Mac OS X
 
-For now, the easiest way to get going is to install Linux on a virtual
-machine, and follow the [Linux instructions](./linux.html).
+The tools you need are similar to the tools listed in the [Linux instructions](./linux.html), but we will need to build a cross compiler in order to get things working. This is sort of complicated and boring, but we've done some of that boring work for you so you can get up and running quickly.
 
-Otherwise, to install the other tools on Mac OS X, make sure to have [homebrew](http://brew.sh/) installed.
+Make sure you have [homebrew](http://brew.sh/) installed because you are going to need it to install some of the tools. You are probably also going to need [Xcode](https://developer.apple.com/xcode/download/) if you don't already have it.
```

Comment:
Yeah good point. I remember having those same Xcode issues. I'll update this section probably this weekend.